### ISSUE TITLE
[BUG] Error - Your Site is Experiencing a Technical Issue

### ISSUE THREAD

**Issue by courtchatter** [Highfivery/zero-spam-for-wordpress#340]
**Describe the bug**

I upgraded the Zero Spam plugin to Version 5.4. and recxeived the Email Your Site is Experiencing a Technical Issue

Given what the email said I have deactivated Zero Spam until I am notified that the error is fixed. 

WordPress caught an error with one of your plugins, Zero Spam for WordPress.

Here are the reported error details (I have removed my site name from the details):

WordPress caught an error with one of your plugins, Zero Spam for WordPress.

the page where the error was caught /wp-admin/admin-ajax.php

WordPress version 6.0.2
Active theme: MH Magazine (version 3.9.13)
Current plugin: Zero Spam for WordPress (version 5.4.3)
PHP version 7.4.30

Error Details
=============
An error of type E_ERROR was caused in line 127 of the file /wp-content/plugins/zero-spam/modules/davidwalsh/class-davidwalsh.php. Error message: Uncaught ArgumentCountError: Too few arguments to function ZeroSpam\Modules\DavidWalsh\DavidWalsh::settings(), 1 passed in /wp-includes/class-wp-hook.php on line 307 and exactly 2 expected in /wp-content/plugins/zero-spam/modules/davidwalsh/class-davidwalsh.php:127
Stack trace:
#0 /wp-includes/class-wp-hook.php(307): ZeroSpam\Modules\DavidWalsh\DavidWalsh->settings(Array)
#1 /wp-includes/plugin.php(191): WP_Hook->apply_filters(Array, Array)
#2 /wp-content/plugins/zero-spam/core/class-settings.php(430): apply_filters('zerospam_settin...', Array)
#3 /wp-content/plugins/zero-spam/modules/davidwalsh/class-davidwalsh.php(35): ZeroSpam\Core\Settings::get_settings('davidwalsh')
#4 /wp-includes/class-wp-hook.php(307): ZeroSpam\Modules\DavidWalsh\DavidWalsh->init('')
 
**To Reproduce**

Error details given in the email are reported above

**Expected behavior**
The plugin should not cause the issue.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Edge
 - Version Version 105.0.1343.42 (Official build) (64-bit)

**Additional context**

**Comment by bmarshall511** [comment]
@courtchatter We're unable to replicate the issue & the line it's complaining about doesn't reflect what's in the current version's code. Can you ensure you're running the latest version? If still experiencing it, can you provide a staging site where we can replicate it (with credentials to admin & code)?

**Comment by courtchatter** [comment]
As stated on the Initial Bug Error details
WordPress version 6.0.2
Active theme: MH Magazine (version 3.9.13)
Current plugin: Zero Spam for WordPress (version 5.4.3)
PHP version 7.4.30

Sorry but no I am unable to provide a staging environment though I can provide a support user access with Admin rights. I'm not sure how best to provide that to you as email replies would get reproduced here and for obvious reasons I would not want to publish a username and password with admin rights.

What I had done was upgraded from the latest version of 5.3 to 5.4.3 
My logic was to see if version 5.4 had any bug reports before making the upgrade from 5.3

Having done the plugin upgrade I began looking at each of the features listed in the General tab. It was when I went to the Error Log that it said there was a Technical Issue --- An email was then received detailing the issue and that is what I have provided to you.
